src/libcollections/str.rs

```diff
@@ -1494,7 +1472,7 @@ impl str {
     /// ```
     /// let s = "Löwe 老虎 Léopard";
     ///
-    /// assert_eq!(s.find(|c: char| c.is_whitespace()), Some(5));
+    /// assert_eq!(s.find(cahr::is_whitespace), Some(5));
```

Comment:
Sorry, it's fixed now. I meant only to ping you once the tests passed.

You may have noticed that this is one example of removing some more needless usage of closures.